### ISSUE TITLE
Add "disabled" state to "Select" component

### ISSUE THREAD

**Issue by diondiondion** [5app/base5-ui#114]
The Select component should be able to be disabled, for when changing the selected value needs to be prevented.

Need to investigate how to do this in an accessible way; maybe `aria-disabled` & disabling all menu interactions will be enough?
